pkg/onload/control_plane.go

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
+package onload
```

Comment:
(little) nit: Could the package name be a little more specific? the `cmd/` directory is called `worker`
I just think that using the generic "onload" name might give a false impression that this package is a golang implementation / integration with onload rather than something that just configures cplane parameters.

Reply:
Let's call it `control_plane`, which is a precise and also uninviting name!